src/breadcrumb-group/__tests__/breadcrumb-group.test.tsx

```diff
@@ -69,15 +69,7 @@ describe('BreadcrumbGroup Component', () => {
 
     test('has ellipsis', () => {
       expect(wrapper.findDropdown()!.findNativeButton()).not.toBe(null);
-      expect(wrapper.findByClassName(styles.ellipsis)!.getElement()).toHaveAttribute('aria-label', 'Show path');
-    });
-
-    test('can set aria-label for ellipsis', () => {
```

Comment:
Now that we have aria-label set to a button we can confirm it with a dedicated test as well.